app-i18n/translate-shell/translate-shell-0.9.6.11.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Online command-line translator"
+HOMEPAGE="https://www.soimort.org/translate-shell/"
+SRC_URI="https://github.com/soimort/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Unlicense"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+bidi +curl test tts"
+
+RDEPEND="
+	app-misc/rlwrap
+	sys-apps/gawk
+	curl? ( net-misc/curl[ssl] )
```

Comment:
These use flags doesn't seem to be doing anything. USE-controlled rdepends is forbidden per https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Policies#USE-Controlled_Optional_RDEPENDs

If they have no function during configure or compile, they should be moved to `optfeature` or similar instead.